src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/Footer.vue

```diff
@@ -1,22 +1,31 @@
 <template>
   <v-footer app absolute :style="{ 'min-height': '50px' }">
     <v-container>
-      <div class="d-flex w-100 justify-space-between">
-        <router-link to="/">
-          <img src="../assets/bc-gov-logo.png" width="155" class="logo" alt="B.C. Government Logo" />
-        </router-link>
-        <div class="w-lg-25 w-50">
+      <AcknowledgementBanner
```

Comment:
To make the Acknowledgement banner match designs and span full width on mobile I think we will need to keep it seperate from the `<Footer />` component. 

I.e.

```
<EceAcknowledgmentBanner />
<EceFooter />
```

Reply:
I moved the acknowledgement banner outside the component so it spans the entire width.